components/admin-panel/sections/ActivityLog/ActivityTypeFilter.js

```diff
@@ -3,85 +3,157 @@ import PropTypes from 'prop-types';
+      'COLLECTIVE_EXPENSE_PAID',
+      'COLLECTIVE_EXPENSE_MARKED_AS_UNPAID',
+      'COLLECTIVE_EXPENSE_MARKED_AS_SPAM',
+      'COLLECTIVE_EXPENSE_MARKED_AS_INCOMPLETE',
+      'COLLECTIVE_EXPENSE_PROCESSING',
+      'COLLECTIVE_EXPENSE_SCHEDULED_FOR_PAYMENT',
+      'COLLECTIVE_EXPENSE_ERROR',
+      'COLLECTIVE_EXPENSE_INVITE_DRAFTED',
+      'COLLECTIVE_EXPENSE_RECURRING_DRAFTED',
+      'COLLECTIVE_EXPENSE_MOVED',
+      'EXPENSE_COMMENT_CREATED',
+      'TAXFORM_REQUEST',
+    ],
   },
-  {
-    value: 'EXPENSES',
-    type: 'EXPENSES',
-    label: defineMessage({ id: 'Expenses', defaultMessage: 'Expenses' }),
+  ACCOUNTS: {
+    title: defineMessage({ defaultMessage: 'Accounts' }),
+    activities: [
+      'COLLECTIVE_CREATED',
+      'COLLECTIVE_EDITED',
+      'COLLECTIVE_FROZEN',
+      'COLLECTIVE_UNFROZEN',
+      'COLLECTIVE_CONTACT',
+      'CONNECTED_ACCOUNT_CREATED',
+    ],
   },
-  {
-    value: 'CONTRIBUTIONS',
-    type: 'CONTRIBUTIONS',
-    label: defineMessage({ id: 'Contributions', defaultMessage: 'Contributions' }),
+  CONTRIBUTIONS: {
+    title: defineMessage({ id: 'Contributions', defaultMessage: 'Contributions' }),
+    activities: [
+      'SUBSCRIPTION_CANCELED',
+      'SUBSCRIPTION_ACTIVATED',
+      'SUBSCRIPTION_CONFIRMED',
+      'CONTRIBUTION_REJECTED',
+      'TICKET_CONFIRMED',
+      'ORDER_CANCELED_ARCHIVED_COLLECTIVE',
+      'ORDER_PROCESSING',
+      'ORDER_PROCESSING_CRYPTO',
+      'ORDER_PENDING_CONTRIBUTION_NEW',
+      'ORDER_THANKYOU',
+      'ORDERS_SUSPICIOUS',
+      'PAYMENT_FAILED',
+    ],
   },
-  {
-    value: 'UPDATES',
-    type: 'ACTIVITIES_UPDATES',
-    label: defineMessage({ defaultMessage: 'Updates & Conversations' }),
+  UPDATES: {
+    title: defineMessage({ defaultMessage: 'Updates & Conversations' }),
+    activities: [
+      'COLLECTIVE_UPDATE_CREATED',
+      'COLLECTIVE_UPDATE_PUBLISHED',
+      'UPDATE_COMMENT_CREATED',
+      'CONVERSATION_COMMENT_CREATED',
+      'COLLECTIVE_CONVERSATION_CREATED',
+    ],
   },
-  {
-    value: 'VIRTUAL_CARDS',
-    type: 'VIRTUAL_CARDS',
-    label: defineMessage({ id: 'VirtualCards.Title', defaultMessage: 'Virtual Cards' }),
+  MEMBERS: {
+    title: defineMessage({ defaultMessage: 'Members' }),
+    activities: [
+      'COLLECTIVE_MEMBER_INVITED',
+      'COLLECTIVE_CORE_MEMBER_ADDED',
+      'COLLECTIVE_CORE_MEMBER_INVITED',
+      'COLLECTIVE_CORE_MEMBER_INVITATION_DECLINED',
+      'COLLECTIVE_CORE_MEMBER_REMOVED',
+      'COLLECTIVE_CORE_MEMBER_EDITED',
+    ],
   },
-  {
-    value: 'GIFT_CARDS',
-    type: ['USER_CARD_INVITED', 'USER_CARD_CLAIMED'],
-    label: defineMessage({ id: 'editCollective.menu.giftCards', defaultMessage: 'Gift Cards' }),
+  VIRTUAL_CARDS: {
+    title: defineMessage({ id: 'VirtualCards.Title', defaultMessage: 'Virtual Cards' }),
+    activities: [
+      'COLLECTIVE_VIRTUAL_CARD_SUSPENDED',
+      'COLLECTIVE_VIRTUAL_CARD_ADDED',
+      'VIRTUAL_CARD_REQUESTED',
+      'VIRTUAL_CARD_CHARGE_DECLINED',
```

Comment:
@Betree : This appears under both `Virtual Cards` and `Contributions` sections. 

[activity-virtual-card.webm](https://user-images.githubusercontent.com/12435965/202560273-7b69fef6-a8fb-4931-a62d-034d78598dc5.webm)